index.html

```diff
@@ -2,56 +2,71 @@
 <html>
   <head>
     <title>Dan Stineback Portfolio</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/icons.css">
     <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="css/modules.css">
+    <script id="projects-template" type="text/x-handlebars-template">
+      <article class="clearfix" data-author="{{developer}}" data-category="{{category}}">
+        <header>
+          <h1>{{title}}</h1>
+          <div class="byline">
+            By {{developer}} <br>
+            <address><a href="{{projectUrl}}" target="_blank">{{title}}</a></address>
+            {{date}}
+          </div>
+        </header>
+        <section class="article-body">{{{body}}}</section>
+        <!-- <a href="#" class="read-on">Read on &rarr;</a> -->
+      </article>
+    </script>
+
   </head>
+  <div class="clearfix"></div>
   <body>
-    <header class="site-header">
+    <header class="site-header clearfix">
+      <section class="heading-group-h1">
         <h1>Dan Stineback Professinal Portfolio</h1>
           <p>Look into the skills of a coding Nija</p>
```

Comment:
Professional, ninja. You want to avoid misspellings on anything you're going to show off.